docs/content/latest/deploy/multi-dc/async-replication.md

```diff
@@ -29,6 +29,8 @@ You can create source and target universes as follows:
 1. Create tables for the APIs being used by the target universe. These should be the same tables as you created for the source universe.
 1. Proceed to setting up [unidirectional](#seting-up-unidirectional-replication) or [bidirectional](#setting-up-bidirectional-replication) replication.
 
+In case if you already have existing data in your tables, then you need to follow the bootstrap process [bootstrapping a sink cluster](#bootstrapping-a-sink-cluster)
+
```

Comment:
```suggestion
{{< note title="Note" >}}
In case you already have existing data in your tables, then you will need to follow the bootstrap process [bootstrapping a sink cluster](#bootstrapping-a-sink-cluster).
{{< /note >}}
```

Reply:
Done